content/en/profiler/profiler_troubleshooting.md

```diff
@@ -232,38 +232,125 @@ For further help with this issue, [contact support][2] and include the output of
+
+   2. Open the `DD-DotNet-Profiler-Native-<Application Name>-<pid>` log file in the `/var/log/datadog` folder.
+
+   3. Look for `libddprof error: Failed to send profile.` entries: it means that it is not possible to contact the agent.
+
+   4. If not present, look for `The profile was sent. Success?` entries.
+
+   5. The following means the profile has been sent successfully:
+      ```
+      true, Http code: 200
+      ```
+
+   6. Check the following other Http codes for error possible reasons:
+      ```
+      false, Http code: XXX
+      ```
+        1. 403:  // TODO
+
+        2. 500:  // TODO
+
+{{% /tab %}}
+
+{{% tab "Windows" %}}
 
 1. Check that the Agent is installed and running and is visible in the Windows Services panel.
 
-2. Check the result of profiles export:
+2. Check the profiler has been loaded from the loader log:
+
+   1. Open the `dotnet-native-loader-<Application Name>-<pid>` log file in the `%ProgramData%\Datadog-APM\logs\DotNet` folder.
+
+   2. Look for `CorProfiler::Initialize: Continuous Profiler initialized successfully.` near the end.
 
-   1. Enable debug logs by setting the `DD_TRACE_DEBUG` environment variable for the application.
+   2. If not present, enable debug logs by setting the `DD_TRACE_DEBUG` environment variable for the application.
 
-   2. Restart the application.
+   3. Restart the application.
 
-   3. Open the `DD-Dotnet-Profiler.<Application Name>` log file in the `%ProgramData%\Datadog-APM\logs\` folder.
+   4. Open the `dotnet-native-loader-<Application Name>-<pid>` log file in the `%ProgramData%\Datadog-APM\logs\DotNet` folder.
 
-   4. Look for `Profile data was NOT successfully exported via HTTP POST` entries.
+   5. Look for the `#Profiler` entry.
 
-   5. Check the following fields for errors:
+   6. Check the following lines to ensure that the profiler library has been loaded:
       ```
-      ["response.StatusCode"]=...,
-      ["response.Error"]="...",
+      [...] #Profiler
+      [...] PROFILER;{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A};win-x64;.\Datadog.Profiler.Native.dll
+      [...] PROFILER;{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A};win-x86;.\Datadog.Profiler.Native.dll
+      [...] DynamicDispatcherImpl::LoadConfiguration: [PROFILER] Loading: .\Datadog.Profiler.Native.dll [AbsolutePath=C:\Program Files\Datadog\.NET Tracer\win-x64\Datadog.Profiler.Native.dll]
+      [...] [PROFILER] Creating a new DynamicInstance object
+      [...] Load: C:\Program Files\Datadog\.NET Tracer\win-x64\Datadog.Profiler.Native.dll
+      [...] GetFunction: DllGetClassObject
+      [...] GetFunction: DllCanUnloadNow
       ```
 
-   6. Check the following field to ensure that the right URL is used. If you use default configuration settings:
+3. Check the result of profiles export:
+
+   1. If not set already, enable debug logs by setting the `DD_TRACE_DEBUG` environment variable for the application and restart it.
```

Comment:
```suggestion
   1. Ensure debug logs are enabled by setting the `DD_TRACE_DEBUG` environment variable to `true` for the application and restart it.
```

Reply:
Same update

1. If debug logs were not enabled in step 2.2, set the `DD_TRACE_DEBUG` environment variable to `true` for the application and restart it.